### ISSUE TITLE
[Feature] Repeat/loop option

### ISSUE THREAD

**Issue by vrchmvgx** [calzoneman/sync#914]
## FEATURE REQUEST:  Repeating videos ##

**Please confirm whether you've tried the following debugging steps:**

N/A
  
### Description of the Problem ###

  - We use cytube to manage music playlists for tabletop RPG sessions
  - Often, songs/videos are too short that the DM must either interrupt themselves to restart tracks or queue up several instances
  - A repeat option to repeat the currently playing track does not, on surface level, sound like a major addition but would be very useful for usecases like ours

### System Information ###

N/A

**Comment by calzoneman** [comment]
If you uncheck "Add as temporary" when adding a video, or if you later click the "make permanent" button, the video will not be removed from the playlist after playing, and the playlist will loop.  Is that what you meant?

**Comment by vrchmvgx** [comment]
Ah, no, we do that. I'm referring more to, when having a playlist of multiple tracks, having the ability to "repeat one", that is, when video 5 ends, set the next video to 5 again if the loop button is on.

(I am extremely not good at JS, and don't know your codebase, but in essence I'm suggesting/requesting a toggle between `next_video_index = current_video_index + 1 % list_length` and `next_video_index = current_video_index`)

E: https://github.com/calzoneman/sync/blob/8562b2173e5b404921b2475196cc1857ce63c357/src/channel/playlist.js#L1186
So basically, the result if repeat/loop is on would be just calling `this.startPlayback()` without any changes to `this.current`.
